### ISSUE TITLE
Move directories with "F6" DELETES the whole directory

### ISSUE THREAD

**Issue by Artur4ikZX** [coderaiser/cloudcmd#350]
* **Version** (`cloudcmd -v`): v15.9.5
* **Node Version** `node -v`: v16.3.0
* **OS** (`uname -a` on Linux): Docker 5:20.10.73-0debian-buster
* **Browser name/version**: Firefox 89.0
* **Used Command Line Parameters**: 
* **Changed Config**: 

Move and copy SINGLE files works.
Copy and paste directories works also.
But move directories with "F6" do not work, it ends up with the Error, and it DELETES the whole directory into nirvana.

```
ENOENT: no such file or directory, lstat '/mnt/fs/srv/dev-disk-by-uuid-78e15d64-30d0-47fe-a600-cded67a22f08/NAS01/Serien/SG/StarGate Continuum.mkv' Promise {
  <rejected> [Error: ENOENT: no such file or directory, lstat '/mnt/fs/srv/dev-disk-by-uuid-78e15d64-30d0-47fe-a600-cded67a22f08/NAS01/Serien/SG/StarGate Continuum.mkv'] {
    errno: -2,
    code: 'ENOENT',
    syscall: 'lstat',
    path: '/mnt/fs/srv/dev-disk-by-uuid-78e15d64-30d0-47fe-a600-cded67a22f08/NAS01/Serien/SG/StarGate Continuum.mkv'
  }
}
cloudcmd --terminal: The argument 'id' must be a non-empty string. Received ''
url: http://localhost:8000/
```

**Comment by coderaiser** [comment]
Could you please tell me are you moving files located in same device, or different volumes `/dev/sda1` `/dev/sda2`?

**Comment by Artur4ikZX** [comment]
> 
> 
> Could you please tell me are you moving files located in same device, or different volumes `/dev/sda1` `/dev/sda2`?

Moving files in the same volume, like `/dev/sda1` `/dev/sda1` works fine.
It does not work between devices like `/dev/sda1` and `/dev/sdb1`
Unlikely, I can't check what it does between volumes on one device.

**Comment by coderaiser** [comment]
Thanks a lot! 🐞  Fixed with:
- https://github.com/cloudcmd/move-files/commit/9145973fa2881c39583b2066a980c7f772e6ff0f [v5.0.0](https://github.com/cloudcmd/move-files/releases/tag/v5.0.0)
- https://github.com/coderaiser/redzip/commit/313d88bdf2294aff9e20d401dd581f2f969238ba [v2.1.4](https://github.com/coderaiser/redzip/blob/master/ChangeLog#L1-L4)

Landed in [v15.9.6](https://github.com/coderaiser/cloudcmd/releases/tag/v15.9.6) 🎉

Is it works for you?

**Comment by Artur4ikZX** [comment]
> 
> 
> Thanks a lot! 🐞 Fixed with:
> 
>     * [cloudcmd/move-files@9145973](https://github.com/cloudcmd/move-files/commit/9145973fa2881c39583b2066a980c7f772e6ff0f) [v5.0.0](https://github.com/cloudcmd/move-files/releases/tag/v5.0.0)
> 
>     * [coderaiser/redzip@313d88b](https://github.com/coderaiser/redzip/commit/313d88bdf2294aff9e20d401dd581f2f969238ba) [v2.1.4](https://github.com/coderaiser/redzip/blob/master/ChangeLog#L1-L4)
> 
> 
> Landed in [v15.9.6](https://github.com/coderaiser/cloudcmd/releases/tag/v15.9.6) 🎉
> 
> Is it works for you?

yep it works! thx!
